packages/sitecore-jss-dev-tools/src/codegen/extract-components.ts

```diff
@@ -0,0 +1,76 @@
+import { ExtractedFileType, resolveComponentImportFiles, sendCode } from './utils';
+import { fetchBearerToken } from '../auth/fetch-bearer-token';
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+type ExtractComponentArgs = {
+  appFolder?: string;
+  componentBuilderPath?: string;
+};
+
+/**
+ * Handler for the extract-component API command
```

Comment:
Minor thing, to show description more properly
![image](https://github.com/user-attachments/assets/53babab5-7b9e-4a6f-ba2b-b0bcbef803b3)

```suggestion
 * Handler for the extract-component API process.
```